packages/@lightningjs/ui-components/src/mixins/withExtensions/index.d.ts

```diff
@@ -19,24 +19,28 @@
 import lng from '@lightningjs/core';
 
 export interface WithExtensions {
-  get _withExtensionsApplied(): boolean;
+  _instanceLastThemeUpdateTimestamp?: number;
+
   get _prototypeChain(): Set<string>;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   get _extensions(): Record<string, any>[];
   get _componentExtensions(): unknown[];
-  get _extensionsApplied(): boolean;
 
   _resetComponent(): void;
   _calculateComponentExtensionLength(): number;
   _createExtension(): void;
-  _createExtensionClass(): unknown;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  _createExtensionAliases(obj: Record<string, any>): object;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  _setComponentAliases(aliasObj: Record<string, any>): void;
```

Comment:
We've been adding `@deprecated` to a comment block above things we've removed to warn users that they're no longer supported: https://github.com/rdkcentral/Lightning-UI-Components/blob/6f6c1dfd6d0a056fe41f0aa95ca63a6ab2040861/packages/%40lightningjs/ui-components/src/components/Tile/Tile.d.ts#L34